tests/integrated/test-restart-io/runtest

```diff
@@ -51,7 +51,7 @@ except FileExistsError:
     pass
 
 with DataFile(
-    os.path.join(restartdir, "BOUT.restart.0.nc"), create=True
+    os.path.join(restartdir, "BOUT.restart.0.nc"), create=True, format="NETCDF3_64BIT"
```

Comment:
Do we still need this with the update to boutdata?

Reply:
I think I fixed the issue: https://github.com/boutproject/boututils/pull/47